.github/workflows/builddocumentation.yml

```diff
@@ -0,0 +1,20 @@
+name: Build
+
+on:
+    pull_request:
+      branches: [ 'main', 'master' ]
+      types:
+      - opened
+      - synchronize
+      - reopened
+      - closed
+      paths:
+      - 'docs/**'
+env:
+    http_proxy: http://proxy-chain.intel.com:912
```

Comment:
```suggestion
    http_proxy: http://proxy-chain.intel.com:911
```

Reply:
this file is copy paste from their examples.

Reply:
https://intelpedia.intel.com/Intel_Proxy_servers

Reply:
Link says 911 for http & 912 for https

Reply:
yes, so port for **http_proxy** is 911 and port for **https_proxy** is 912